tasks/main.yml

```diff
@@ -18,10 +18,11 @@
 - name: Install Docker's dependencies
   apt:
     name: "{{ docker__package_dependencies + docker__pip_dependencies }}"
+    force_apt_get: true
```

Comment:
Can you remove this?

This seems to be an Ansible bug that will be addressed in future 2.8.x patches: https://github.com/ansible/ansible/issues/56832